grails-app/services/org/pih/warehouse/inventory/OutboundStockMovementService.groovy

```diff
@@ -148,6 +152,11 @@ class OutboundStockMovementService {
                     'in'("shipmentType", shipmentTypes)
                 }
             }
+        }
+
+        // Get result count
```

Comment:
"Get result count" is not accurate I'd say, because we also return stock movements from there, not only "total count" (which I think you meant here).

Here I'd add a short note: "This query is about to return data with filters applied from the query above, but this will not filter out any approvers from the result. See the comment above".